VMBackup/main/Utils/SizeCalculation.py

```diff
@@ -68,11 +69,15 @@ def __init__(self,patching, hutil, logger,para_parser):
             self.logger.log("As the LunList is empty including all disks")
     
     def get_lsscsi_list(self):
-        command = "sudo lsscsi"
+        if (self.sudo_off == 1):
```

Comment:
nit: it could be written as following too.
command = "lsscsi"
if(self.sudo_off == 0)
     command = "sudo " + command